src/components/Sidebar/cart/CartSection.js

```diff
@@ -23,26 +25,50 @@ export const styles = {
   },
 };
 
-function CartSection({ classes, section }) {
-  let leftHeaderContent;
-  if (section.schedule.length > 1) {
-    leftHeaderContent = section.schedule
-      .map(schedule => getFormattedClassSchedule(schedule))
-      .reduce(
-        (resultStr, formattedStr) => `${resultStr}, ${formattedStr}`,
-      );
-  } else {
-    leftHeaderContent = getFormattedClassSchedule(section.schedule[0]);
+class CartSection extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showDialog: false,
+    };
+    this.toggleDialog = this.toggleDialog.bind(this);
+  }
+
+  toggleDialog() {
+    this.setState(state => ({ showDialog: !state.showDialog }));
+  }
+
+  render() {
+    const { classes, section } = this.props;
+    const { showDialog } = this.state;
```

Comment:
Super nitpicky thing, but let's put a space between the prop/state destructures and the rest of the render. Common pattern.